frontend/src/metabase/dashboard/context/context.tsx

```diff
@@ -189,29 +184,6 @@ const DashboardContextProviderInner = ({
     [onError],
   );
 
-  const fetchId = useCallback(
-    async (idToFetch: DashboardId) => {
-      try {
-        const { id, isError } = await dispatch(
-          fetchEntityId({ type: "dashboard", id: idToFetch }),
-        );
```

Comment:
`/dashboard/:id` supports an entity id now, so no need for frontend translation.